config_parser/config_test.go

```diff
@@ -0,0 +1,83 @@
+package config_parser
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_ParseConfig(t *testing.T) {
+	testCases := []struct {
```

Comment:
I suggest adding a tests that hit every block of `ParseConfig`. For example, test for:
- an empty file path;
- non-canonical inputs (unquoted strings and integers in place of strings, this is likely going to happen).